graphql/ui_plugin_build_baron.go

```diff
@@ -193,16 +195,24 @@ func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnIn
 	bbProj, ok := buildBaronProjects[t.Project]
 
 	if !ok {
-		return nil, bbConfig, errors.Errorf("Build Baron project for %s not found", t.Project)
+		// build baron project not found, meaning it's not configured for
+		// either regular build baron or for a custom ticket filing webhook
+		return nil, bbConfig, nil
 	}
 	bbConfig.ProjectFound = true
 
 	// the build baron is configured if the jira search is configured
 	if len(bbProj.TicketSearchProjects) <= 0 {
 		bbConfig.SearchConfigured = false
-		return nil, bbConfig, errors.Errorf("Build Baron ticket search projects for %s not found", t.Project)
+		return nil, bbConfig, nil
 	}
 	bbConfig.SearchConfigured = true
+
+	// if the project uses custom ticket creation, don't use the default search
+	if bbProj.TaskAnnotationSettings.FileTicketWebHook.Endpoint != "" {
+		return nil, bbConfig, errors.Errorf("Build Baron project for %s is configured to use a custom webhook", t.Project)
```

Comment:
Should this return an error or just not return the default search? I'm a little confused about what's information and what's an error

Reply:
I thought about it a bit more. Although the intention was for these to be separate, if someone specifically wants to use the build baron for search and a custom ticket for create, then there is no reason to stop them. 
I will add another check in build_baron.go to make sure they can't double up on the ticket creation.